raspbian/applications/camera.md

```diff
@@ -273,10 +273,10 @@ DRC changes the images by increasing the range of dark areas, and decreasing the
 By default, DRC is off.
 
 ```
-	--stats,	-st		Display image statistics
+	--stats,	-st		Image statistics use stills capture
```

Comment:
"Image statistics use stills capture" sounds like a nonsense-phrase?

Would something like "Use current-frame statistics for stills capture" be technically correct? :shrug:

Reply:
Made change which probably makes more sense.